src/number_theory/padics/padic_norm.lean

```diff
@@ -48,7 +48,7 @@ namespace padic_norm
 
 section padic_norm
 open padic_val_rat
-variables (p : ℕ)
+variables {p : ℕ}
 
 /-- Unfolds the definition of the p-adic norm of `q` when `q ≠ 0`. -/
 @[simp] protected lemma eq_zpow_of_nonzero {q : ℚ} (hq : q ≠ 0) :
```

Comment:
It's not clear to me that making `p` implicit is a good idea here. The usual rule is that if `p` doesn't appear in any hypotheses (which it doesn't), then it should be explicit.

Reply:
Sure, but that's the whole point of this PR. My justification is that once you've chosen a prime `p` (e.g. when defining something with `padic_val_rat`, etc), there won't be many use cases where you want to switch to a different `p` to apply some lemma, and they can somehow infer this `p`. The very few use cases can simply use `@`.

Reply:
This looks like a reasonable change to me. Should we merge it?

Reply:
@riccardobrasca I have reverted all the changes requested, so this is just awaiting someone to approve.

Reply:
There is a building error, can you please fix it?